.github/workflows/beam_PreCommit_Whitespace.yml

```diff
@@ -19,10 +19,10 @@ on:
   push:
     tags: ['v*']
     branches: ['master', 'release-*']
-    paths: ['*.md', '*.build.gradle','.github/workflows/beam_PreCommit_Whitespace.yml']
+    paths: ['*.md', '*.gradle', '*.gradle.kts', '.github/workflows/beam_PreCommit_Whitespace.yml']
```

Comment:
This should in sync with https://github.com/apache/beam/blob/df0c25d5c315baa880355c4cf5499135f1aa4998/sdks/python/test-suites/tox/py38/build.gradle#L152